examples/testprograms/http/http.star

```diff
@@ -3,11 +3,12 @@ load("@http", "myhttp")
 def foo(x):
     return "meow, " + x
 
-def on_http_get():
+def on_http_get(conn):
```

Comment:
in AWS they pass (event, context) to any lambda
context can have multiple fields in it and is extendable 
event usually mean a string of the event that triggered this lambda

maybe we should have something similar since multiple events can point to the same entrypoint and additional information like the connection (if exists) and be added to the context

another option is just context and setting everything inside it

Reply:
We need to formally define how events are passed to event handlers. We can borrow from serverless:
- In AWS it's `handler(event, context)`
- In Google it's `handler(request)`
- In OpenFass it's `handler(request)`
- ...

Currently in Python the handler gets a single `event` where the trigger data is in a field called `data`